etc/openqa/openqa.ini

```diff
@@ -78,6 +78,8 @@
 ## Causes jobs reported as incomplete by the worker to be cloned automatically when the
 ## reason matches; set to 0 to disable
 #auto_clone_regex = ^(cache failure: |terminated prematurely: |api failure: Failed to register .* 503|backend died: .*VNC.*(timeout|timed out|refused)|QEMU terminated: Failed to allocate KVM HPT of order 25.* Cannot allocate memory)
+## How many times should a job be restarted
+#auto_clone_limit = 20;
```

Comment:
I don't think there should be semicolon here

Reply:
semicolon removed